### ISSUE TITLE
Error on https://www.onlinemictest.com/games/mic-frog/

### ISSUE THREAD

**Issue by JSkes** [ruffle-rs/ruffle#14051]
# Error Info
Error name: Error
Error message: panicked at /home/runner/.cargo/git/checkouts/wgpu-53e70f8674b08dd4/49d16f7/wgpu-core/src/resource.rs:207:13:
Map callback was leaked
Error stack:
```
Error: panicked at /home/runner/.cargo/git/checkouts/wgpu-53e70f8674b08dd4/49d16f7/wgpu-core/src/resource.rs:207:13:
Map callback was leaked
    at imports.wbg.__wbg_new_ab87fd305ed9004b (chrome-extension://donbcfbmhbcapadipfkeojnmajbakjdc/dist/339.js:2624:21)
    at core::ops::function::FnOnce::call_once{{vtable.shim}}::h0ac237062e7987e0 (wasm://wasm/033f8df2:wasm-function[9988]:0x9cdb73)
    at std::panicking::rust_panic_with_hook::h3aa054d35a0817d7 (wasm://wasm/033f8df2:wasm-function[6557]:0x8cdb63)
    at core::panicking::panic_fmt::h3e1dd3d08288569e (wasm://wasm/033f8df2:wasm-function[9785]:0x9c88ac)
    at <wgpu_core::resource::BufferMapCallback as core::ops::drop::Drop>::drop::hc8ab1557643976cd (wasm://wasm/033f8df2:wasm-function[9139]:0x992b5f)
    at <T as wgpu::context::DynContext>::surface_configure::h1e61bb12248351a5 (wasm://wasm/033f8df2:wasm-function[8280]:0x94f89e)
    at wgpu::Surface::configure::hbec3d3339959a789 (wasm://wasm/033f8df2:wasm-function[4183]:0x7a3f47)
    at <ruffle_render_wgpu::target::SwapChainTarget as ruffle_render_wgpu::target::RenderTarget>::resize::h2d810531aa643395 (wasm://wasm/033f8df2:wasm-function[10412]:0x9e39a0)
    at <ruffle_render_wgpu::backend::WgpuRenderBackend<T> as ruffle_render::backend::RenderBackend>::set_viewport_dimensions::hb1a0a8ab773c5500 (wasm://wasm/033f8df2:wasm-function[5198]:0x853a69)
    at ruffle_core::player::Player::set_viewport_dimensions::h3fad2ca265de9c8a (wasm://wasm/033f8df2:wasm-function[1494]:0x4a0995)
```

# Player Info
Allows script access: true
Renderer: wgpu
Adapter Backend: Gl
Adapter Name: "ANGLE (NVIDIA, NVIDIA GeForce GTX 960 (0x00001401) Direct3D11 vs_5_0 ps_5_0, D3D11)"
Adapter Device Type: Other
Adapter Driver Name: ""
Adapter Driver Info: ""
Enabled features: Features(TEXTURE_COMPRESSION_BC | TEXTURE_ADAPTER_SPECIFIC_FORMAT_FEATURES | PUSH_CONSTANTS | SHADER_UNUSED_VERTEX_OUTPUT)
Available features: Features(CLEAR_TEXTURE | MULTIVIEW)
Current limits: Limits { max_texture_dimension_1d: 16384, max_texture_dimension_2d: 16384, max_texture_dimension_3d: 2048, max_texture_array_layers: 256, max_bind_groups: 4, max_bindings_per_bind_group: 1000, max_dynamic_uniform_buffers_per_pipeline_layout: 8, max_dynamic_storage_buffers_per_pipeline_layout: 0, max_sampled_textures_per_shader_stage: 16, max_samplers_per_shader_stage: 16, max_storage_buffers_per_shader_stage: 0, max_storage_textures_per_shader_stage: 0, max_uniform_buffers_per_shader_stage: 11, max_uniform_buffer_binding_size: 16384, max_storage_buffer_binding_size: 0, max_vertex_buffers: 8, max_buffer_size: 268435456, max_vertex_attributes: 16, max_vertex_buffer_array_stride: 255, min_uniform_buffer_offset_alignment: 256, min_storage_buffer_offset_alignment: 256, max_inter_stage_shader_components: 60, max_compute_workgroup_storage_size: 0, max_compute_invocations_per_workgroup: 0, max_compute_workgroup_size_x: 0, max_compute_workgroup_size_y: 0, max_compute_workgroup_size_z: 0, max_compute_workgroups_per_dimension: 0, max_push_constant_size: 96 }
Surface quality: high
Surface samples: 4
Surface size: Extent3d { width: 550, height: 400, depth_or_array_layers: 1 }
Player type: Object
SWF URL: /wp-content/themes/onlinemictest/assets-micfrog/Pond.swf
Param movie: https://www.onlinemictest.com/wp-content/themes/onlinemictest/assets-micfrog/Pond.swf
Param quality: high
Param bgcolor: #cccccc
Param play: true
Param loop: true
Param wmode: direct
Param scale: showall
Param menu: true
Param devicefont: false
Param allowScriptAccess: sameDomain
Attribute 0: undefined
Attribute 1: undefined
Attribute 2: undefined
Attribute 3: undefined
Attribute 4: undefined
Attribute 5: undefined

# Page Info
Page URL: https://www.onlinemictest.com/games/mic-frog/
SWF URL: https://www.onlinemictest.com/wp-content/themes/onlinemictest/assets-micfrog/Pond.swf

# Browser Info
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.0.0 Safari/537.36
Platform: Win32
Has touch support: false

# Ruffle Info
Version: 0.1.0
Name: nightly 2023-11-16
Channel: nightly
Built: 2023-11-16T00:14:37.965Z
Commit: ffdb248872c89b586a83aaefc5cca3c98b8e5e8c
Is extension: true

# Metadata
width: 850
height: 850
frameRate: 60
numFrames: 2
swfVersion: 14
backgroundColor: #0000FF
isActionScript3: true
uncompressedLength: 3310389

**Comment by n0samu** [comment]
The problem happens when entering or exiting fullscreen.

**Comment by n0samu** [comment]
Fixed by #13732
